tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature

```diff
@@ -434,3 +434,81 @@ Feature: upload file
+    When user "Brian" uploads file "filesForUpload/zerobyte.txt" to "Shares/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Brian" the content of the file "/texftile.txt" of the space "Shares" should be ""
+    And the content of file "/textfile.txt" for user "Alice" should be ""
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+
+  @skipOnReva @issue-7257
+  Scenario: user replaces a shared file with an empty file using spaces dav path
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "file with content" to "/textfile.txt"
+    And user "Alice" has shared file "/textfile.txt" with user "Brian" with permissions "read,update"
+    When user "Brian" uploads a file "filesForUpload/zerobyte.txt" to "/textfile.txt" in space "Shares" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Brian" the content of the file "/textfile.txt" of the space "Shares" should be ""
+    And the content of file "/textfile.txt" for user "Alice" should be ""
+
+  @skipOnReva @issue-7257
+  Scenario: user replaces a file inside a project space with empty file
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "file with content" to "/textfile.txt"
+    When user "Alice" uploads a file "filesForUpload/zerobyte.txt" to "/textfile.txt" in space "new-space" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
+
+  @skipOnReva @issue-7257
+  Scenario: user replaces a file inside a shared space with empty file
+    Given using spaces DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "file with content" to "/textfile.txt"
+    And user "Alice" has shared a space "new-space" with settings:
+      | shareWith | Brian  |
+      | role      | editor |
+    When user "Brian" uploads a file "filesForUpload/zerobyte.txt" to "/textfile.txt" in space "new-space" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Brian" the content of the file "/textfile.txt" of the space "new-space" should be ""
+    And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
```

Comment:
These scenarios behave the same as the above scenarios added in https://github.com/owncloud/ocis/pull/8088. So I think we should not add these as duplication.

@PrajwolAmatya Could you remove these new scenarios and add only tags change in this PR?